controllers/account.go

```diff
@@ -139,7 +139,7 @@ func (c *ApiController) Signup() {
 		invitationName = invitation.Name
 	}
 
-	if application.IsSignupItemVisible("Email") && application.GetSignupItemRule("Email") != "No verification" && authForm.Email != "" {
```

Comment:
but in some case, Email or Phone number is not required, we should consider those situation

Reply:
Doesn't `application.GetSignupItemRule("Email") != "No verification"` check for this?
It can be either `No verification` or `Normal` in UI

Reply:
ok